src/index.js

```diff
@@ -12,7 +12,7 @@ import {
   RPCHandlersToken,
   ResponseError,
 } from 'fusion-plugin-rpc';
-import {createRPCReducer} from 'fusion-rpc-redux';
+import {createRPCReducer, ActionType} from 'fusion-rpc-redux';
```

Comment:
This should be `import type {ActionType}`

Reply:
Done